src/vector/nalgebra_serial.rs

```diff
@@ -33,6 +35,38 @@ impl<'a, T: Scalar> VectorView<'a> for DVectorView<'a, T> {
     fn into_owned(self) -> Self::Owned {
         self.into_owned()
     }
+    fn scalar_mul(&self, rhs: Self::T) -> Self::Owned {
+        self * rhs
+    }
+}
+impl<'a, T: Scalar> Mul<Scale<T>> for DVectorView<'a, T> {
```

Comment:
same with faer_serial, use macros to reduce the repeated code

Reply:
The same here, I managed to create one macro, the other functions are different enough.

Reply:
ok, if a macro doesn't work here then don't worry about it.

Reply:
I might be able to create a more generic macro but it will take me some more time, I'll create a reminder to come back here later. I want to implement Matrix first